src/it/scala/org/zalando/spearheads/innkeeper/AcceptanceSpecsHelper.scala

```diff
@@ -29,7 +31,7 @@ object AcceptanceSpecsHelper extends ScalaFutures {
       .futureValue
   }
 
-  def postSlashRoutes(routeType: String)(token: String, routeName: String): HttpResponse = {
+  def postSlashRoutes(routeType: String)(routeName: String, token: String = ""): HttpResponse = {
```

Comment:
Plus all the other fns in this file tend to take the token as a first argument.. So you are also inconsistent... :-1: